app/https.ts

```diff
@@ -0,0 +1,21 @@
+import { redirect } from '@remix-run/node';
+
+export const enforceHttps = (request: Request) => {
+  let url = new URL(request.url);
+  let hostname = url.hostname;
+  let proto = request.headers.get('X-Forwarded-Proto') ?? url.protocol;
```

Comment:
Neither hostname nor proto change. Should they be `const`? Also should proto be named `protocol`?